amr-wind/incflo_advance.cpp

```diff
@@ -332,6 +333,8 @@ void incflo::ApplyPredictor (bool incremental_projection)
         icns().solve(dt_diff);
     }
 
+    amr_wind::field_ops::field_lower_bound(repo().get_field("tke"), 1.0e-15);
+        
```

Comment:
We need a better way to handle this. Currently, this assumes we are always running with some PDE for TKE equation system.